src/bundle/Resources/views/Security/base.html.twig

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8" />
+        <title>eZ Platform</title>
+        {% stylesheets filter='scssphp'
```

Comment:
I'd expect it to share some parts with `layout.html.twig` instead of duplicating the code.